CMakeLists.txt

```diff
@@ -246,7 +267,13 @@ else()
 endif(BUILD_STATIC_LIB)
 
 if(MSVC)
-    set_target_properties(_lightgbm PROPERTIES OUTPUT_NAME "lib_lightgbm")
+  set_target_properties(_lightgbm PROPERTIES OUTPUT_NAME "lib_lightgbm")
+  # TODO: use uniquely named dll (in setup.py, lightgbm/libpath.py) to avoid runtime conflicts
+  #if(__OPENCL_PYTHON_PACKAGE)
+  #  set_target_properties(_lightgbm PROPERTIES OUTPUT_NAME "lib_lightgbm-${OPENCL_PYTHON_PACKAGE_STAMP}")
+  #else()
+  #  set_target_properties(_lightgbm PROPERTIES OUTPUT_NAME "lib_lightgbm")
+  #endif()
```

Comment:
can you please remove this commented code?

Reply:
Removed.